diff --git a/poppler/GlobalParams.cc b/poppler/GlobalParams.cc
index 01975901..4b7c0b72 100644
--- a/poppler/GlobalParams.cc
+++ b/poppler/GlobalParams.cc
@@ -98,6 +98,18 @@
 
 std::unique_ptr<GlobalParams> globalParams;
 
+#if defined(ENABLE_RELOCATABLE)
+#pragma warning("ENABLE_RELOCATABLE defined")
+#else
+#pragma warning("ENABLE_RELOCATABLE NOT defined")
+#endif
+
+#if defined(_WIN32)
+#pragma warning("_WIN32 defined")
+#else
+#pragma warning("_WIN32 NOT defined")
+#endif
+
 #if defined(ENABLE_RELOCATABLE) && defined(_WIN32)
 
 /* search for data relative to where we are installed */
@@ -127,6 +139,8 @@ static const char *get_poppler_datadir(void)
 
     unsigned char *p;
 
+    printf("get_poppler_datadir()\n");
+
     if (beenhere)
         return retval;
 
@@ -143,6 +157,7 @@ static const char *get_poppler_datadir(void)
     strcat(retval, "\\share\\poppler");
 
     beenhere = 1;
+    printf("%s\n", retval);
 
     return retval;
 }
